qemu/tests/blockdev_commit_fio.py

```diff
@@ -17,7 +17,7 @@ def fio_thread(self):
             logging.info("Start to run fio")
             fio = generate_instance(self.params, self.main_vm, 'fio')
             try:
-                fio.run(fio_options)
+                fio.run(fio_options, timeout=2400)
```

Comment:
this timeout may not be useful, for we set runtime=1800 for fio, i.e. fio will be running 1800s at most.

Reply:
I have met fio run timeout(1800s) issue if only extending the timeout value in 'job_utils.wait_until_block_job_completed()' during the test on ppc64le, so there might be some race condition that will trigger the timeout altough the fio runtime is also 1800s.

Reply:
thanks for your information, yes, the session will be timed out for the default session alive time is 600s